content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -105,11 +152,36 @@ To encrypt the data:
          - secrets
        providers:
          - kms:
-             name: myKmsPlugin
+             name: myKmsPluginFoo
+             endpoint: unix:///tmp/socketfile.sock
+             cachesize: 100
+             timeout: 3s
+         - kms:
+             name: myKmsPluginBar
+             endpoint: unix:///tmp/socketfile.sock
+             cachesize: 100
+             timeout: 3s
+   ```
+  **KMS v2**
```

Comment:
I'd use an actual heading here, and set a fragment identifier similar to the other examples. Or you could use tabs here.